app/controllers/lti_deployments_controller.rb

```diff
@@ -149,7 +149,7 @@ def choose_course
     if request.post?
       begin
         course = Course.find(params[:course])
-        unless Instructor.exists?(user: real_user, course: course)
+        unless current_user.admin_user? || Instructor.exists?(user: real_user, course: course)
```

Comment:
We should probably turn this into a policy. I see you added one course policy down below. I wonder if there should be a more generic `manage_lti_deployments?` policy under `CoursePolicy` that we can use for all types of management.